apps/condo/domains/common/utils/codegeneration/generate.server.utils.js

```diff
@@ -99,11 +113,32 @@ function generateServerUtils (gql) {
         })
     }
 
+    // getOrCreate
+
+    /**
+     * Updates and returns either existing item or creates new one and returns it
+     * @param context
+     * @param attrs
+     * @param existingItem
+     * @returns {Promise<*|null|undefined>}
+     */
+    async function createOrUpdate (context, attrs, existingItem) {
```

Comment:
This fuction is not hellp a lot. It's better to use an inline code for such cases like so: `(existingItem && existingItem.id) ? Obj.update() : Obj.create()`. Like so: https://docs.djangoproject.com/en/4.0/ref/models/querysets/#get-or-create

Reply:
look at this: https://github.com/open-condo-software/condo/pull/1051/files#r787657016

Reply:
fixed